src/Context/FeatureContext.php

```diff
@@ -963,6 +962,9 @@ public function create_config( $subdir = '', $extra_php = false ) {
 
 		$params['skip-salts'] = true;
 
+		// Do not check database connection as we might be running SQLite and the check would fail.
```

Comment:
@swissspidy Where does the database check happen otherwise?

Reply:
This argument is passed to the `wp config create` command further down, which does this database check by default:

https://github.com/wp-cli/config-command/blob/9de6ce3536a2db56ae5264c61b6f1a35e9132e01/src/Config_Command.php#L159-L171

With SQLite there is not really a point in doing such a check.

An argument could be made for only doing such a check here if `dbtype === mysql`, but given this is for the test suite, I don't see a need for it.

Reply:
I'd prefer for the check to be disabled conditionally. Otherwise, we're actually never checking whether our checks properly checks... ;)
If we'd introduce a bug into the DB check upon config creation, we'd never notice it during the test suite run.

Reply:
Done in 32e9f5e24275315d8a4e28d5dd11e24adef53a2f